File: C/3039.c
#include <stdio.h>

int main()
{
    char str[10], gen;
    int qtd, carrinhos=0, bonecas=0;
    scanf("%d", &qtd);
    for(int i=1; i<=qtd; i++){
        scanf("%s %c", str, &gen);
     
        if(gen == 'M'){
            carrinhos++;
        } else{
            bonecas++;
        }
    }
    
    printf("%d carrinhos\n%d bonecas\n", carrinhos, bonecas);
    return 0;
}

File: Python/3039.py
carrinhos = 0
bonecas = 0
numero_criancas = int(input())

for i in range(numero_criancas):
    entrada = input().split()
    sexo = entrada[1]
    if sexo == 'M':
        carrinhos += 1
    else:
        bonecas += 1

print(f"{carrinhos} carrinhos\n{bonecas} bonecas")
